app/src/main/java/com/hammer/talkbbokki/presentation/topics/TopicListScreen.kt

```diff
@@ -48,111 +114,131 @@ fun TopicListScreen(modifier: Modifier = Modifier, onClickToDetail: (id : String
 
     val currentOffset = currentIndex.value + (offset.value / itemWidth)
 
-    var isVisible by remember { mutableStateOf(false) }
-
     val cardList = (0..10).toList()
-    BoxWithConstraints {
-        var index = ""
-
-        // 대화 주제 리스트
-        LazyRow(
-            state = listState,
-            modifier = modifier.fillMaxSize(),
-            flingBehavior = snapFlingBehavior,
-            verticalAlignment = Alignment.CenterVertically,
-            contentPadding = PaddingValues(
-                start = ((maxWidth - 120.dp) / 2),
-                end = ((maxWidth - 120.dp) / 2)
-            )
+
+    Column() {
+        Spacer(modifier = Modifier.weight(1f))
+        BoxWithConstraints(
+            modifier = Modifier.weight(9f)
         ) {
-            itemsIndexed(cardList) { definiteIndex, item ->
-                index = currentOffset.roundToInt().toString()
-                CardItems(definiteIndex, currentOffset)
+            // 대화 주제 리스트
+            LazyRow(
+                state = listState,
+                modifier = Modifier.fillMaxSize(),
+                flingBehavior = snapFlingBehavior,
+                verticalAlignment = Alignment.CenterVertically,
+                contentPadding = PaddingValues(
+                    start = ((maxWidth - (dimensionResource(id = R.dimen.card_width))) / 2),
+                    end = ((maxWidth - (dimensionResource(id = R.dimen.card_width))) / 2)
+                )
+            ) {
+                itemsIndexed(cardList) { definiteIndex, _ ->
+                    onFocusedCardChange(currentOffset.roundToInt().toString())
+                    CardItem(definiteIndex, currentOffset)
+                }
             }
         }
+    }
+}
 
-        Button(
-            onClick = { /*isVisible = isVisible.not()*/
-                onClickToDetail(index)
-            },
-            modifier = modifier.align(Alignment.BottomCenter)
-        ) {
-            Text("카드 선택")
-        }
+@Composable
+fun SelectBtn(viewCnt: Int, onCardClicked: () -> Unit, modifier: Modifier) {
+    Button(
+        colors = ButtonDefaults.buttonColors(backgroundColor = Color.Black),
+        onClick = {
+            if (viewCnt >= 3)
+            //TODO : 광고 시청 후 디테일로 넘어가는 로직
```

Comment:
🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (12) (should be 16)